app/serializers/submission_serializer/sections/means_details.rb

```diff
@@ -31,6 +39,10 @@ def to_builder
       def income
         @income ||= crime_application.income
       end
+
+      def outgoings
+        @outgoings ||= crime_application.outgoings
```

Comment:
Does this method need to default to an empty array something like:

```
@outgoings ||= (crime_application.outgoings || [])
```

Or will `crime_application.outgoings` always be `not nil`

Reply:
The schema will happily accept it as nil values, so I chose to handle that on each attr with the safe navigation operator. 
My understanding is that if they have gone down the means route, and have got an `income` for which there is a guard clause at the top of this file, then they will have an outgoings